wasm-node/javascript/src/public-types.ts

```diff
@@ -177,6 +177,14 @@ export interface Chain {
      */
     nextJsonRpcResponse(): Promise<string>;
 
+    /**
+     * JSON-RPC responses or notifications to be generated.
+     *
+     * @throws {@link JsonRpcDisabledError} If the JSON-RPC system was disabled in the options of the chain.
+     * @throws {@link CrashError} If the background client has crashed.
+     */
+    jsonRpcResponses: AsyncIterableIterator<string>
```

Comment:
What about?

```suggestion
    readonly jsonRpcResponses: AsyncIterableIterator<string>
```

And turn `jsonRcpResponses` into a getter?

Reply:
I have made this a `readonly` property. I think a property make more sense compared to a getter, because there's in fact only 1 iterable here:
- Each iteration yield the next value
- If multiple iterables are created and are iterated on, they will actually overlap (similar to call `nextJsonResponse` simultaniously)

Reply:
What I meant by getter is to do `get jsonRpcResponses() { return { next: ..., ... } }`

Reply:
Right, here I believe just a TypeScript `readonly` modifier is sufficient, like technically speaking other properties/methods like `sendJsonRpc`, `nextJsonRpcResponse`, etc are actually mutable as well. If you want to enforce immutability at runtime on the JS side, I think calling `Object.freeze` on the entire `chain` object is more effective.